devel/build

```diff
@@ -19,7 +19,14 @@ export GIT_REVISION=$(git describe --tags --abbrev=40 --always --dirty || true)
 # Calling `docker run nextstrain/base` will still only pull down the small base
 # image rather than pulling down the larger nextstrain/base-builder image.
 
-docker build \
+platform=linux/amd64
+
+# `buildx create` is necessary to use a driver that supports multi-platform
+# images.
+docker buildx create --use
```

Comment:
Ah, this will create a new builder every time it's run, which is ok for the fresh envs of a CI, but not for local dev.

Reply:
@tsibley how about b96b606625360eed3a18abde2238c3aff58f8f7b?

Reply:
I pushed a fixup commit that's a variant of that to remove the builder in more cases: 06f7c593917280f7278cdf950413b0c3701a5a15